package.json

```
@@ -26,13 +27,19 @@
   },
   "homepage": "https://github.com/flitto/eslint-config-flitto-typescript#readme",
   "dependencies": {
-    "eslint-config-love": "^47.0.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.1.3"
+    "eslint-config-flitto-typescript": "^1.3.1",
+    "eslint-config-love": "^87.0.0",
+    "eslint-config-prettier": "^10.0.1",
+    "eslint-plugin-prettier": "^5.2.3",
+    "globals": "^15.14.0"
   },
   "devDependencies": {
-    "@types/jest": "^29.5.12",
-    "jest": "^29.7.0",
-    "ts-jest": "^29.1.2"
+    "@types/chai": "^5.0.1",
+    "@types/eslint": "^9.6.1",
+    "@types/mocha": "^10.0.10",
+    "@types/node": "^22.10.7",
+    "chai": "^5.1.2",
+    "mocha": "^11.0.1",
+    "tsx": "^4.19.2"
   }
```

```suggestion
  },
      "license": "MIT",
      "engines": {
        "node": ">=18"
      },
```
모듈 올라간걸 봣더니, 18부터를 진원하는게 많아진거 같은데요. 
저희도 최소버전을 정해두는건 어떨까 한번 건의드립니다.  🙇‍♂️